gcc/rust/backend/rust-compile-intrinsic.h

```diff
@@ -18,6 +18,7 @@
 #define RUST_COMPILE_INTRINSIC
 
 #include "rust-compile-context.h"
```

Comment:
does it need the include here?

Reply:
You're right, I cleaned it up to include only the essentials. Good point